clang.cindex.LibclangError: /usr/lib/llvm-15/lib/libclang.so: cannot open shared object file: No such file or directory. To provide a path to libclang use Config.set_library_path() or Config.set_library_file().

It seems problem started happening again:
```
Found ninja-1.11.1.git.kitware.jobserver-1 at /usr/local/bin/ninja
ninja: Entering directory `/home/runner/work/rizin/rizin/rz-bindgen/build'
[1/7] Generating bindgen_outputs with a custom command
FAILED: rizin.i 
/usr/bin/python3 ../src/main.py -o . --clang-path /usr/lib/llvm-15/lib --clang-args ' -resource-dir=/usr/lib/llvm-14/lib/clang/14.0.0' --rizin-include-path /usr/include/librz --targets SWIG
Traceback (most recent call last):
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 4174, in get_cindex_library
    library = cdll.LoadLibrary(self.get_filename())
  File "/usr/lib/python3.10/ctypes/__init__.py", line [45](https://github.com/rizinorg/rizin/actions/runs/6296209026/job/17091351925?pr=3888#step:7:46)2, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /usr/lib/llvm-15/lib/libclang.so: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/rizin/rizin/rz-bindgen/build/../src/main.py", line 51, in <module>
    bindings.run()
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/bindings.py", line 45, in run
    for translation_unit, builder, func in zip(
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 621, in result_iterator
    yield _result_or_cancel(fs.pop())
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 319, in _result_or_cancel
    return fut.result(timeout)
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 451, in result
    return self.__get_result()
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/cparser_header.py", line 123, in translation_unit
    return TranslationUnit.from_source(
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 2813, in from_source
    index = Index.create()
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 2700, in create
    return Index(conf.lib.clang_createIndex(excludeDecls, 0))
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 212, in __get__
    value = self.wrapped(instance)
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 41[48](https://github.com/rizinorg/rizin/actions/runs/6296209026/job/17091351925?pr=3888#step:7:49), in lib
    lib = self.get_cindex_library()
  File "/home/runner/work/rizin/rizin/rz-bindgen/src/clang/cindex.py", line 41[79](https://github.com/rizinorg/rizin/actions/runs/6296209026/job/17091351925?pr=3888#step:7:80), in get_cindex_library
    raise LibclangError(msg)
clang.cindex.LibclangError: /usr/lib/llvm-15/lib/libclang.so: cannot open shared object file: No such file or directory. To provide a path to libclang use Config.set_library_path() or Config.set_library_file().
[2/7] Generating plugin/swig_runtime with a custom command
ninja: build stopped: subcommand failed.
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/local/bin/ninja -C /home/runner/work/rizin/rizin/rz-bindgen/build
Error: Process completed with exit code 1.
```

It seems that Ubuntu Jammy uses outdated LLVM 14:
```
Get:9 http://azure.archive.ubuntu.com/ubuntu jammy/universe amd64 libclang-dev amd64 1:14.0-55~exp2 [3138 B]
Get:10 http://azure.archive.ubuntu.com/ubuntu jammy/universe amd64 llvm-runtime amd64 1:14.0-55~exp2 [3204 B]
Get:11 http://azure.archive.ubuntu.com/ubuntu jammy/universe amd64 llvm amd64 1:14.0-55~exp2 [3758 B]
```

--- a/meson.build
+++ b/meson.build
@@ -19,7 +19,7 @@ target_sphinx = targets.contains('sphinx')
 doxygen_path = get_option('doxygen_path')
 
 if clang_path == ''
-  llvm_config = find_program('llvm-config', required: false)
+  llvm_config = find_program('llvm-config', 'llvm-config-7', required: false)
   if llvm_config.found()
     clang_path = run_command(llvm_config, '--libdir', check: true).stdout().strip()
   elif build_machine.system() == 'darwin'

--- a/meson.build
+++ b/meson.build
@@ -19,9 +19,9 @@ target_sphinx = targets.contains('sphinx')
 doxygen_path = get_option('doxygen_path')
 
 if clang_path == ''
-  llvm = dependency('llvm', required: false)
-  if llvm.found()
-    clang_path = llvm.get_variable(cmake: 'LLVM_LIBRARY_DIRS', configtool: 'libdir', default_value: 'none')
+  llvm_config = find_program('llvm-config', required: false)
+  if llvm_config.found()
+    clang_path = run_command(llvm_config, '--libdir', check: true).stdout().strip()
   elif build_machine.system() == 'darwin'
     clang_path = '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/'
   elif build_machine.system() == 'windows'

--- a/src/bindings.py
+++ b/src/bindings.py
@@ -425,7 +425,7 @@ def bind_ls(ls_h: Header) -> None:
     Class(ls_h, typedef="SdbList")
 
 
-@threaded_header("sdb/ht_pp.h")
+@threaded_header("rz_util/ht_pp.h")
 def bind_ht_pp(ht_pp_h: Header) -> None:
     """
     ht_pp
@@ -433,7 +433,7 @@ def bind_ht_pp(ht_pp_h: Header) -> None:
     Class(ht_pp_h, typedef="HtPP")
 
 
-@threaded_header("sdb/ht_pu.h")
+@threaded_header("rz_util/ht_pu.h")
 def bind_ht_pu(ht_pu_h: Header) -> None:
     """
     ht_pu
@@ -441,7 +441,7 @@ def bind_ht_pu(ht_pu_h: Header) -> None:
     Class(ht_pu_h, typedef="HtPU")
 
 
-@threaded_header("sdb/ht_up.h")
+@threaded_header("rz_util/ht_up.h")
 def bind_ht_up(ht_up_h: Header) -> None:
     """
     ht_up
@@ -449,7 +449,7 @@ def bind_ht_up(ht_up_h: Header) -> None:
     Class(ht_up_h, typedef="HtUP")
 
 
-@threaded_header("sdb/ht_uu.h")
+@threaded_header("rz_util/ht_uu.h")
 def bind_ht_uu(ht_uu_h: Header) -> None:
     """
     ht_uu

--- a/src/cparser_header.py
+++ b/src/cparser_header.py
@@ -98,12 +98,6 @@ class HeaderBuilder:
 
         assert rizin_include_path
         filename = os.path.abspath(os.path.join(rizin_include_path, *name_segments))
-
-        # Fix sdb path if using source librz/include
-        if name_segments[0] == "sdb" and not os.path.exists(filename):
-            name_segments = ["..", "util", "sdb", "src"] + name_segments[1:]
-            filename = os.path.abspath(os.path.join(rizin_include_path, *name_segments))
-
         assert os.path.exists(filename)
         self.filename = filename
 

